core/templates/dev/head/pages/exploration_editor/feedback_tab/feedback_tab.html

```diff
@@ -104,8 +100,7 @@ <h2>Exploration Feedback</h2>
 
     <hr>
 
-    {% if user_email %}
-      <div class="row">
+      <div class="row" ng-if="userIsLoggedIn">
```

Comment:
I guess you missed here :p
(Indentation)

Reply:
oops... sorry

Reply:
fixed it